File: lab_3/small-primes.R
#!/usr/bin/Rscript

n <- as.numeric(readLines('stdin'))

if (n >= 2 && (n == 2 || n %% 2 != 0) && (n == 3 || n %% 3 != 0) && (n == 5 || n %% 5 != 0)){
  print('prime')
} else  {
  print('not prime')
}

File: Lab 3/small-primes.py
#!/usr/bin/env python3

n = int(input())

if n < 2:
    print('not prime')
elif n == 2:
    print('prime')
elif n % 2 == 0 or (n % 3 == 0 and n != 3):
    print('not prime')
else:
    print('prime')
